src/manage_nsfs/manage_nsfs_glacier.js

```diff
@@ -135,6 +136,28 @@ async function run_glacier_operation(fs_context, log_namespace, cb) {
     }
 }
 
+/**
+ * force_gpfs_fs_context returns a shallow copy of given
+ * fs_context with backend set to 'GPFS'.
+ *
+ * NOTE: The function will throw error if it detects that libgfs
+ * isn't loaded.
+ *
+ * @param {nb.NativeFSContext} fs_context
+ * @returns {nb.NativeFSContext}
+ */
+function force_gpfs_fs_context(fs_context) {
```

Comment:
since this doesn't always force a gpfs backend (see last case), perhaps better rename to prepare_galcier_fs_context